plugins/module_utils/repo_setup/get_hash/__main__.py

```diff
@@ -20,12 +20,6 @@
 import sys
 from repo_setup.utils import load_logging
 from repo_setup.get_hash.hash_info import HashInfo
-import repo_setup.get_hash.exceptions as exc
-
-
-def _validate_args(parsed_args):
-    if parsed_args.os_version == "centos7" and (parsed_args.component is not None):
-        raise exc.HashInvalidParameter("Cannot specify component for centos 7")
```

Comment:
If you submit another commit, please remove the comment on line 37:
"This is NOT valid for Centos 7."